### PR TITLE
Mo' Unit Tests

### PR DESCRIPTION
**Pull request by ChocoChipset**

### CODE REVIEW

File: Tests/SPTPersistentCacheDebugUtilitiesTests.m [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+@implementation SPTPersistentCacheDebugUtilitiesTests
+
+- (void)testNilDebugCallback
+{
+    XCTAssertNoThrow(SPTPersistentCacheSafeDebugCallback(@"",
+                                                         nil),
+                     @"A nil callback shouldn't cause an exception.");
+}
+
```
Comment (rastersize) [inline Tests/SPTPersistentCacheDebugUtilitiesTests.m:37]: Could we add a positive test as well? E.g.

``` objc
__block NSString *callbackMessage = nil;
SPTPersistentCacheSafeDebugCallback(@"the message", ^(NSString *message){
    callbackMessage = message;
});

XCTAssertEqualObjects(callbackMessage, @"the message", @"The callback should be executed and pass the message to the callback");
```

?

**Comment by rastersize** [comment]
:+1: 

Extra :star: for a positive debug test.

File: Tests/SPTPersistentCacheDebugUtilitiesTests.m (hunk 1 above)
Comment (ChocoChipset) [inline Tests/SPTPersistentCacheDebugUtilitiesTests.m:37]: Yeah, Good thinking!

Comment (rastersize) [inline Tests/SPTPersistentCacheDebugUtilitiesTests.m:37]: Nice gif, I’m saving that one :floppy_disk:

**Comment by rastersize** [comment]
:+1:

**Pull request merged**